plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java

```diff
@@ -456,7 +459,7 @@ public void keyPressed(KeyEvent e) {
 		});
 		EditingUtils.setID(username, EsbViewsRepository.OAuthMediator.Properties.username);
 		EditingUtils.setEEFtype(username, "eef::Text"); //$NON-NLS-1$
-		FormUtils.createHelpButton(widgetFactory, parent, propertiesEditionComponent.getHelpContent(EsbViewsRepository.OAuthMediator.Properties.username, EsbViewsRepository.FORM_KIND), null); //$NON-NLS-1$
+		FormUtils.createHelpButton(widgetFactory, filterCredentialsSubPropertiesGroup, propertiesEditionComponent.getHelpContent(EsbViewsRepository.OAuthMediator.Properties.username, EsbViewsRepository.FORM_KIND), null); //$NON-NLS-1$
```

Comment:
Limit line length to 120 characters.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/806